include/shellstrings.h

```diff
@@ -41,7 +53,18 @@ void shell_intro();
  * Returns:
  *  - None.
  */
-void shell_save();
+void shell_save(bool *color);
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
changed!